File: main/src/main/java/org/anibyl/slounik/network/ArticlesLoader.java
package org.anibyl.slounik.network;

import android.content.Context;

/**
 * Articles loader interface.
 * <p/>
 * Created by Usievaład Kimajeŭ on 23.12.15.
 */
public interface ArticlesLoader {
    void loadArticles(String wordToSearch, Context context, ArticlesCallback callBack);
}

File: main/src/main/java/org/anibyl/slounik/network/ArticlesLoader.kt
package org.anibyl.slounik.network

import android.content.Context

/**
 * Articles loader interface.
 *
 * @author Usievaład Kimajeŭ
 * @created 23.12.2015
 */
interface ArticlesLoader<in T> where T : ArticlesCallback {
	fun loadArticles(wordToSearch: String, context: Context, callback: T)
}
